src/content/1.7/development/webservice/_index.md

```diff
@@ -195,6 +195,61 @@ Here is what a resource API call could look like (in this case `http://example.c
+* `language` to control which language values are returned
+
+##### Control returned fields with "display"
+
+The `display` parameter can be used to return all fields when used with the **full** value: `http://example.com/api/addresses/1?display=full`.
+
+You can also ask for certain fields if you use a list of fields names in brackets: `http://example.com/api/addresses/1?display=[id,lastname,firstname,phone_mobile]`
+
+{{% notice note %}}
+A response obtained with "display" other than "full" can't be used in a **PUT** (update) request, because the `WebserviceRequest` class validation for fields is the same for **POST** (create) and **PUT** (update).
+
+This should be fixed in a near future with a *yet-to-come-pull-request*!
+{{% /notice %}}
+
+##### Control returned items with "filter"
+
+The **EQUAL** operator is used when you need to get specific items. For exemple, if you want the addresses for customer #1, you can filter your **GET** request with the `filter` parameter: `http://example.com/api/addresses?filter[id_customer]=1`
+
+The **LIKE** operator is used when you need to search for items. For exemple, if you want the addresses with cities starting with "SAINT": `http://example.com/api/addresses?filter[city]=[saint]%`
+
+The **AND** operator is used when you need to get items matching several criteria: `http://example.com/api/addresses?filter[city]=[paris|lyon]`
+
+Other operators can be used, such as: 
+
+- **NOT EQUAL** (single value): `http://example.com/api/customers?filter[firstname]=![hubert]` (apologies to all Huberts)
+- **NOT EQUAL** (multiple values): `http://example.com/api/customers?filter[firstname]=![hubert|leon|gaspard]` (apologies again...)
+- **GREATER THAN**: `http://example.com/api/customers?filter[birthday]=>[2000-00-00%2000:00:00]` (millenials only :-))
+- **LOWER THAN**: `http://example.com/api/customers?filter[birthday]=<[2000-00-00%2000:00:00]` (previous century only :-D)
+
+This can be used in combination with the `display` parameter! Let's say you want to get the mobile phone numbers of customers #1, #7 and #42: `http://example.com/api/addresses?filter[id_customer]=[1|7|42]&display=[phone_mobile]`
+
+You can also filter by dates! A typical example would be a routine in an ERP fetching the orders since the last call: `http://example.com/api/orders?display=full&date=1&filter[date_add]=[2019-11-14%2013:00:00,2019-11-14%2014:00:00]`. In this exemple, we request the orders created on 2019-11-14 between 1pm and 2pm.
+
+{{% notice note %}}
+Pay attention to: 
+
+* The url-encoded space (%20) in the datetime values
+* The `date=1` parameter used to allow date filtering
+* The dates range, with an inclusive first member and an exclusive last member (from 13:00:00 to 13:59:59)
+{{% /notice %}}
+
+##### Special parameters
+
+The `date=1` parameter must be used to allow date filtering (see exemple above).
+
+The `limit=0,100` parameter can be used to limit the number of returned items (similar to MySQL's LIMIT clause).
+
```

Comment:
There's also `sort=[field1_ASC,field2_DESC]` which you can use to sort the results.